lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/util/LibertyConstants.java

```diff
@@ -28,7 +28,7 @@ private LibertyConstants() {
 
     public static final String PUBLIC_VISIBILITY = "PUBLIC";
 
-    public static final String WLP_USER_CONFIG_DIR = File.separator + String.join(File.separator, "usr", "shared", "config") + File.separator;
-    public static final String SERVER_CONFIG_DROPINS_DEFAULTS = File.separator + String.join(File.separator, "configDropins", "defaults") + File.separator;
-    public static final String SERVER_CONFIG_DROPINS_OVERRIDES = File.separator + String.join(File.separator, "configDropins", "overrides") + File.separator;
+    public static final String WLP_USER_CONFIG_DIR = "/usr/shared/config/";
```

Comment:
Add a comment here explaining why it is ok, and in fact required, that the forward slash is used here.

Reply:
The original intent of `File.separator` was to cover the slash differences between operating systems. With #152, it's revealed that backslash appears specifically on Windows paths and not URIs. If the backslash is a minor case and gets addressed with a `.replace`, we can make forward slash the standard.
This change makes the constants more readable and more direct, over concat and string joins.

Reply:
I meant add a comment to the code. And it can be shorter than what you put here.